### PR TITLE
Typescript (Fix spellchecker issue and bump electron version)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
## Description
Fix issues and add support for BC

### Changes
- Fixed spellchecker module
- Bumped up electron version to 5.x
- Update unit test case
- Updated app menu to support 5.x
- Fixed protocol handler

### Added
- Demo content for language changes
- Demo content for protocol handler

## QA Checklist
- [X] Unit-Tests
- [ ] Automation-Tests

### CODE REVIEW

File: src/app/crypto-handler.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { app } from 'electron';
```
Comment (VishwasShashidhar) [inline src/app/crypto-handler.ts]: we don't need native encryption anymore. So, we might as well remove this.

File: src/app/protocol-handler.ts [hunk 2]
```diff
@@ -61,7 +61,7 @@ class ProtocolHandler {
      */
     public processArgv(argv?: string[]): void {
         const protocolUriFromArgv = getCommandLineArgs(argv || process.argv, protocol.SymphonyProtocol, false);
-        if (isWindowsOS && protocolUriFromArgv) {
+        if (protocolUriFromArgv) {
```
Comment (VishwasShashidhar) [inline src/app/protocol-handler.ts]: reason for this change please?

**Comment by VishwasShashidhar** [comment]
@KiranNiranjan Please rebase with master as well.

File: src/app/crypto-handler.ts (hunk 1 above)
Comment (KiranNiranjan) [reply src/app/crypto-handler.ts]: Native encryption work if we disable `contextIsolation`

File: src/app/protocol-handler.ts (hunk 2 above)
Comment (KiranNiranjan) [reply src/app/protocol-handler.ts]: We need to handle `protocol handler` from the cmd line for both `Windows` & `Mac` so removed this check

**Comment by KiranNiranjan** [comment]
@VishwasShashidhar It's up to date with the master

**Pull request merged**